taker-frontend/src/components/Trade.tsx

```diff
@@ -221,8 +222,20 @@ export default function Trade({
                                     </Td>
                                 </Tr>
                                 <Tr>
-                                    <Td>Perpetual Cost</Td>
-                                    <Td isNumeric>{fundingRateHourly ? fundingRateHourly + "%" : "not available"}</Td>
+                                    <Td><Text>Perpetual Cost</Text></Td>
+                                    <Tooltip
+                                        label={`The CFD is rolled over perpetually every hour at ${fundingRateHourly}%, annualized that is ${fundingRateAnnualized}%. 
+                                        The funding rate can fluctuate depending on the market movements.`}
+                                        hasArrow
+                                        placement={"right"}
+                                        isDisabled={!fundingRateHourly}
+                                    >
+                                        <Td isNumeric>
+                                            {fundingRateHourly
+                                                ? "Hourly @ " + fundingRateHourly + "%"
+                                                : "not available"}
```

Comment:
Seen this pattern a couple of times. You could either use a `Skeleton` here or make your own component that displays "not available" if the value is undefined.

Reply:
good idea. Changed to using a `skeleton`